pkg/types/health.go

```diff
@@ -15,8 +15,8 @@ import (
 const (
 	// DefaultSnapshotLeaseRenewalEnabled is a default value for enabling the snapshot lease renewal feature
 	DefaultSnapshotLeaseRenewalEnabled = false
-	// DefaultFullSnapshotLeaseUpdateInterval is a default interval for updating full snapshot lease
-	DefaultFullSnapshotLeaseUpdateInterval = 3 * time.Minute
+	// FullSnapshotLeaseUpdateInterval is a default interval for updating full snapshot lease
```

Comment:
```suggestion
	// FullSnapshotLeaseUpdateInterval is a interval for updating full snapshot lease
```
It is no longer a "default" value as it is no longer configurable.

Reply:
Done